packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java

```diff
@@ -257,6 +257,11 @@ public B customHeaders(Map<String, String> customHeaders) {
 			this.customHeaders = customHeaders;
 			return self();
 		}
+
+		public B httpClient(final HttpClient httpClient) {
+			this.httpClient = httpClient;
+			return self();
+		}
```

Comment:
I spent some effort on allowing the `HttpClient` to be dependency injected, for the purposes of unit testing this feature, but ultimately did not end up using it. Still seems like it could be a useful feature to someone down the line to I kept it.